.github/workflows/build.yml

```
@@ -67,6 +67,14 @@ jobs:
       - name: Copy mobility to artifacts
         run: cp target/thumbv8m.main-none-eabihf/release/dis-bootloader artifacts/dis-bootloader-mobility.elf
 
+      - name: Build turing
+        uses: actions-rs/cargo@v1
+        with:
+          command: build
+          args: --release --features turing
+      - name: Copy mobility to artifacts
```

Please change the name

```suggestion
      - name: Copy turing to artifacts
```